app/jobs/hackney/income/jobs/send_green_in_arrears_msg_job.rb

```diff
@@ -28,6 +32,12 @@ def green_in_arrears_sms_template_id
         def green_in_arrears_email_template_id
           Rails.configuration.x.green_in_arrears.email_template_id
         end
+
+        def check_expiration
+          current_job = Delayed::Job.where('handler LIKE ?', "%job_id: #{job_id}%").first
```

Comment:
Would this work instead?
```suggestion
          current_job = Delayed::Job.find(job_id)
```

Reply:
Unfotunatelly `job_id` is inside a YAML field similar to this one:
`--- !ruby/object:ActiveJob::QueueAdapters::DelayedJobAdapter::JobWrapper\njob_data:\n  job_class: Hackney::Income::Jobs::SendGreenInArrearsMsgJob\n  job_id: xxxxxxxxxx \n  provider_job_id: \n  queue_name: message_jobs\n  priority: \n  arguments:\n  - case_id: 3\n    _aj_symbol_keys:\n    - case_id\n  executions: 0\n  locale: en\n`

That's why we need to search for the string "job_id: xxxxx" inside that field.